ova/scripts/systemd/units/dispatch-server.service

```diff
@@ -11,7 +11,7 @@ Restart=on-failure
 RestartSec=10
 
 WorkingDirectory=/var/lib/dispatch
-ExecStart=/usr/bin/dispatch-server --host 0.0.0.0 --enable-tls --tls-certificate /storage/data/certs/server.crt --tls-certificate-key /storage/data/certs/server.key
+ExecStart=/usr/bin/dispatch-server --api-endpoint 172.17.0.1:8080 --host 0.0.0.0 --gateway-tls-port 443 --enable-tls --tls-certificate /storage/data/certs/server.crt --tls-certificate-key /storage/data/certs/server.key
```

Comment:
Is that api-endpoint right?  Will it always be addressible at that IP?

Reply:
At the very least I think it should be given a descriptive variable name so people know what that IP is supposed to be.

Reply:
@berndtj yes, this is the IP of docker0 interface, always set to this IP in photon OVA, reachable from all docker containers. 
@tenczar can you propose some name? I'm honestly at loss here. this is the endpoint that is reachable by event drivers (running as containers).

Reply:
Could we call it DOCKER_0_BRIDGE_IP or something similar?

Reply:
Renamed to events-gateway. Thanks @berndtj !